src/clear.ts

```diff
@@ -0,0 +1,11 @@
+export default function clear(
+  node: Node | null
+): Node | null {
+  if (node) {
```

Comment:
`node.innerHTML =''` might be more efficient?

Reply:
I can see some recent benchmarks stating that removeChild loop is better than innerHTML.
Furthermore, in order to keep supporting IE11 for a while, removeChild loop is cross browser compliant because you can't set innerHTML of a <select> element to an empty string.

https://www.measurethat.net/Benchmarks/Show/6910/0/innerhtml-vs-removechild-vs-remove